platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/events/performance/MemoryMonitor.kt

```diff
@@ -15,10 +15,10 @@ private const val KB = 1024L
 
 internal class MemoryMonitor(
     context: Context,
-) {
+) : MemoryMetricsProvider {
```

Comment:
comment for below: in retrospective I think that computing the memory class value as `activityManager.memoryClass * 1024` was a mistake, since it deviates from that the android sdk gives and what developers expect. I think we should bite the bullet and just change it to be reported in megabytes